docs/attribution/0001-analytics.rst

```diff
@@ -648,49 +648,26 @@ It will also first lint the schema files.
     before being made active in production. Therefore anytime we make new additions
     to these files, those changes should also undergo review.
 
-Using Glean events in individual page bundles
----------------------------------------------
-
-Our analytics code for Glean lives in a single bundle in the base template,
-which is intended to be shared across all web pages. This bundle automatically
-initializes Glean and records page hit events. It also creates some helpers
-that can be used across different page bundles to record interaction events
-such as link clicks and form submissions.
-
-The ``Mozilla.Glean.pageEvent()`` helper can be used to record events that are
-specific to a page, such as successful form completions:
-
-.. code-block:: javascript
-
-    if (typeof window.Mozilla.Glean !== 'undefined') {
-        window.Mozilla.Glean.pageEvent({
-            label: 'newsletter-sign-up-success',
-            type: 'mozilla-and-you' // type is optional
-        });
-    }
-
-It can also be used to record non-interaction events that are not directly
-initiated by a visitor:
+Recording Glean click events
+----------------------------
 
-.. code-block:: javascript
+Glean will automatically record click events on any HTML element that have at least
+one of the following data attributes:
 
-    if (typeof window.Mozilla.Glean !== 'undefined') {
-        window.Mozilla.Glean.pageEvent({
```

Comment:
Note for reviewer: I'm removing mention of `pageEvent` in the docs for now, even though it's still in code. This event will require some future tweaks / updates so that it aligns better with the newer standard glean page_load / click events.